### ISSUE TITLE
after build_app.bat - can't open app 

### ISSUE THREAD

**Issue by MaxNiark** [avalon60/ctk_theme_builder#16]
this is the error message in consol

python theme_builder_setup.py
No pyvenv.cfg file

**Comment by MaxNiark** [comment]
C:\Users\USER>D:\99_PROGRAM_WINDAUBE\Python\Python311\ctk_theme_builder\ctk_theme_builder/ctk_theme_builder.py
Unable to locate database file located at D:\99_PROGRAM_WINDAUBE\Python\Python311\ctk_theme_builder\ctk_theme_builder\assets\data\ctk_theme_builder.db
Traceback (most recent call last):
  File "D:\99_PROGRAM_WINDAUBE\Python\Python311\ctk_theme_builder\ctk_theme_builder\ctk_theme_builder.py", line 14, in <module>
    from lib.control_panel import ControlPanel
  File "D:\99_PROGRAM_WINDAUBE\Python\Python311\ctk_theme_builder\ctk_theme_builder\lib\control_panel.py", line 6, in <module>
    import lib.ctk_theme_builder_m as mod
  File "D:\99_PROGRAM_WINDAUBE\Python\Python311\ctk_theme_builder\ctk_theme_builder\lib\ctk_theme_builder_m.py", line 739, in <module>
    METHOD_LISTENER_ADDRESS = method_listener_address()
                              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\99_PROGRAM_WINDAUBE\Python\Python311\ctk_theme_builder\ctk_theme_builder\lib\ctk_theme_builder_m.py", line 734, in method_listener_address
    _listener_port = listener_port()
                     ^^^^^^^^^^^^^^^
  File "D:\99_PROGRAM_WINDAUBE\Python\Python311\ctk_theme_builder\ctk_theme_builder\lib\ctk_theme_builder_m.py", line 728, in listener_port
    _listener_port = preference_setting(db_file_path=DB_FILE_PATH, scope='user_preference',
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\99_PROGRAM_WINDAUBE\Python\Python311\ctk_theme_builder\ctk_theme_builder\lib\ctk_theme_builder_m.py", line 642, in preference_setting
    raise FileNotFoundError
FileNotFoundError

C:\Users\Mklughertz>

**Comment by MaxNiark** [comment]
![image](https://github.com/avalon60/ctk_theme_builder/assets/126392328/1431e934-f29a-4bb4-b40b-990f6b048b16)
```
C:\Users\USER>python theme_builder_setup.py -i D:\99_PROGRAM_WINDAUBE\Python\Python311\bouh -p D:\99_PROGRAM_WINDAUBE\Python\Python311\bouh/ctk_theme_builder-2.4.0.zip
No pyvenv.cfg file
```

**Comment by avalon60** [comment]
First of all, your installation location looks odd. Are you installing theme builder, where you have Python installed?

Please try an experiment:

```
mkdir d:\apps
D:
cd apps
mkdir ctk_theme_builder
cd ctk_theme_builder
py --version
py -m venv  venv

```
Please copy and paste entire session/output in a reply.

**Comment by avalon60** [comment]
(0, get_1.default)(obj, path); --color-checks-donut-error: var(--color-scale-red-4); --color-checks-donut-pending: var(--color-scale-yellow-3); --color-checks-donut-success: var(--color-scale-green-4); --color-checks-donut-neutral: var(--color-scale-gray-3); --color-checks-dropdown-text: #e6edf3; --color-checks-dropdown-bg: #161b22; --color-checks-dropdown-border: #30363d; --color-checks-dropdown-shadow: rgba(1,4,9,0.3); --color-checks-dropdown-hover-text: #e6edf3; --color-checks-dropdown-hover-bg: rgba(110,118,129,0.1); --color-checks-dropdown-btn-hover-text: #e6edf3; --color-checks-dropdown-btn-hover-bg: rgba(110,118,129,0.1); --color-checks-scrollbar-thumb-bg: rgba(110,118,129,0.4); --color-checks-header-label-text: #7d8590; --color-checks-header-label-open-text: #e6edf3; --color-checks-header-border: #21262d; --color-checks-header-icon: #7d8590; --color-checks-line-text: #7d8590; --color-checks-line-num-text: #6e7681; --color-checks-line-timestamp-text: #6e7681; --color-checks-line-hover-bg: rgba(110,118,129,0.1); --color-checks-line-selected-bg: rgba(56,139,253,0.1); --color-checks-line-selected-num-text: #2f81f7; --color-checks-line-dt-fm-text: #ffffff; --color-checks-line-dt-fm-bg: #9e6a03; --color-checks-gate-bg: rgba(187,128,9,0.15); --color-checks-gate-text: #7d8590; --color-checks-gate-waiting-text: #d29922; --color-checks-step-header-open-bg: #161b22; --color-checks-step-error-text: #f85149; --color-checks-step-warning-text: #d29922; --color-checks-logline-text: #7d8590; --color-checks-logline-num-text: #6e7681; --color-checks-logline-debug-text: #a371f7; --color-checks-logline-error-text: #7d8590; --color-checks-logline-error-num-text: #6e7681; --color-checks-logline-error-bg: rgba(248,81,73,0.1); --color-checks-logline-warning-text: #7d8590; --color-checks-logline-warning-num-text: #d29922; --color-checks-logline-warning-bg: rgba(187,128,9,0.15); --color-checks-logline-command-text: #2f81f7; --color-checks-logline-section-text: #3fb950; --color-checks-ansi-black: #0d1117; --color-checks-ansi-black-bright: #161b22; --color-checks-ansi-white: #b1bac4; --color-checks-ansi-white-bright: #b1bac4; --color-checks-ansi-gray: #6e7681; --color-checks-ansi-red: #ff7b72; --color-checks-ansi-red-bright: #ffa198; --color-checks-ansi-green: #3fb950; --color-checks-ansi-green-bright: #56d364; --color-checks-ansi-yellow: #d29922; --color-checks-ansi-yellow-bright: #e3b341; --color-checks-ansi-blue: #58a6ff; --color-checks-ansi-blue-bright: #79c0ff; --color-checks-ansi-magenta: #bc8cff; --color-checks-ansi-magenta-bright: #d2a8ff; --color-checks-ansi-cyan: #76e3ea; --color-checks-ansi-cyan-bright: #b3f0ff; --color-project-header-bg: var(--color-scale-gray-9); --color-project-sidebar-bg: var(--color-scale-gray-8); --color-project-gradient-in: var(--color-scale-gray-8); --color-project-gradient-out: rgba(22, 27, 34, 0); --color-mktg-btn-bg: #f6f8fa; --color-mktg-btn-shadow-outline: rgba(255, 255, 255, 0.25) 0 0 0 1px inset; --color-mktg-btn-shadow-focus: rgb(255 255 255 / 25%) 0 0 0 4px; --color-mktg-btn-shadow-hover: 0 4px 7px rgba(0, 0, 0, 0.15), 0 100px 80px rgba(255, 255, 255, 0.02), 0 42px 33px rgba(255, 255, 255, 0.024), 0 22px 18px rgba(255, 255, 255, 0.028), 0 12px 10px rgba(255, 255, 255, 0.034), 0 7px 5px rgba(255, 255, 255, 0.04), 0 3px 2px rgba(255, 255, 255, 0.07); --color-mktg-btn-shadow-hover-muted: rgb(255 255 255) 0 0 0 2px inset; --color-control-border-color-emphasis: #606771; --color-avatar-bg: rgba(255,255,255,0.1); --color-avatar-border: rgba(240,246,252,0.1); --color-avatar-stack-fade: #30363d; --color-avatar-stack-fade-more: #21262d; --color-avatar-child-shadow: 0 0 0 2px #0d1117; --color-topic-tag-border: rgba(0,0,0,0); --color-counter-border: rgba(0,0,0,0); --color-select-menu-backdrop-border: #484f58; --color-select-menu-tap-highlight: rgba(48,54,61,0.5); --color-select-menu-tap-focus-bg: #0c2d6b; --color-overlay-shadow: 0 0 0 1px #30363d, 0 16px 32px rgba(1,4,9,0.85); --color-overlay-backdrop: rgba(22,27,34,0.4); --color-header-text: rgba(255,255,255,0.7); --color-header-bg: #161b22; --color-header-divider: #8b949e; --color-header-logo: #f0f6fc; --color-header-search-bg: #0d1117; --color-header-search-border: #30363d; --color-sidenav-selected-bg: #21262d; --color-menu-bg-active: #161b22; --color-input-disabled-bg: rgba(110,118,129,0); --color-timeline-badge-bg: #21262d; --color-ansi-black: #484f58; --color-ansi-black-bright: #6e7681; --color-ansi-white: #b1bac4; --color-ansi-white-bright: #ffffff; --color-ansi-gray: #6e7681; --color-ansi-red: #ff7b72; --color-ansi-red-bright: #ffa198; --color-ansi-green: #3fb950; --color-ansi-green-bright: #56d364; --color-ansi-yellow: #d29922; --color-ansi-yellow-bright: #e3b341; --color-ansi-blue: #58a6ff; --color-ansi-blue-bright: #79c0ff; --color-ansi-magenta: #bc8cff; --color-ansi-magenta-bright: #d2a8ff; --color-ansi-cyan: #39c5cf; --color-ansi-cyan-bright: #56d4dd; --color-btn-text: #c9d1d9; --color-btn-bg: #21262d; --color-btn-border: rgba(240,246,252,0.1); --color-btn-shadow: 0 0 transparent; --color-btn-inset-shadow: 0 0 transparent; --color-btn-hover-bg: #30363d; --color-btn-hover-border: #8b949e; --color-btn-active-bg: hsla(212,12%,18%,1); --color-btn-active-border: #6e7681; --color-btn-selected-bg: #161b22; --color-btn-counter-bg: #30363d; --color-btn-primary-text: #ffffff; --color-btn-primary-bg: #238636; --color-btn-primary-border: rgba(240,246,252,0.1); --color-btn-primary-shadow: 0 0 transparent; --color-btn-primary-inset-shadow: 0 0 transparent; --color-btn-primary-hover-bg: #2ea043; --color-btn-primary-hover-border: rgba(240,246,252,0.1); --color-btn-primary-selected-bg: #238636; --color-btn-primary-selected-shadow: 0 0 transparent; --color-btn-primary-disabled-text: rgba(255,255,255,0.5); --color-btn-primary-disabled-bg: rgba(35,134,54,0.6); --color-btn-primary-disabled-border: rgba(240,246,252,0.1); --color-btn-primary-icon: #ffffff; --color-btn-primary-counter-bg: rgba(4,38,15,0.2); --color-btn-outline-text: #388bfd; --color-btn-outline-hover-text: #58a6ff; --color-btn-outline-hover-bg: #30363d; --color-btn-outline-hover-border: rgba(240,246,252,0.1); --color-btn-outline-hover-shadow: 0 1px 0 rgba(1,4,9,0.1); --color-btn-outline-hover-inset-shadow: inset 0 1px 0 rgba(255,255,255,0.03); --color-btn-outline-hover-counter-bg: rgba(5,29,77,0.2); --color-btn-outline-selected-text: #ffffff; --color-btn-outline-selected-bg: #0d419d; --color-btn-outline-selected-border: rgba(240,246,252,0.1); --color-btn-outline-selected-shadow: 0 0 transparent; --color-btn-outline-disabled-text: rgba(88,166,255,0.5); --color-btn-outline-disabled-bg: #0d1117; --color-btn-outline-disabled-counter-bg: rgba(31,111,235,0.05); --color-btn-outline-counter-bg: rgba(5,29,77,0.2); --color-btn-outline-counter-fg: #388bfd; --color-btn-outline-hover-counter-fg: #58a6ff; --color-btn-outline-disabled-counter-fg: rgba(47,129,247,0.5); --color-btn-danger-text: #f85149; --color-btn-danger-hover-text: #ffffff; --color-btn-danger-hover-bg: #da3633; --color-btn-danger-hover-border: #f85149; --color-btn-danger-hover-shadow: 0 0 transparent; --color-btn-danger-hover-inset-shadow: 0 0 transparent; --color-btn-danger-hover-counter-bg: rgba(255,255,255,0.2); --color-btn-danger-selected-text: #ffffff; --color-btn-danger-selected-bg: #b62324; --color-btn-danger-selected-border: #ff7b72; --color-btn-danger-selected-shadow: 0 0 transparent; --color-btn-danger-disabled-text: rgba(248,81,73,0.5); --color-btn-danger-disabled-bg: #0d1117; --color-btn-danger-disabled-counter-bg: rgba(218,54,51,0.05); --color-btn-danger-counter-bg: rgba(73,2,2,0.2); --color-btn-danger-icon: #f85149; --color-btn-danger-hover-icon: #ffffff; --color-btn-danger-counter-fg: #f85149; --color-btn-danger-hover-counter-fg: #ffffff; --color-btn-danger-disabled-counter-fg: rgba(248,81,73,0.5); --color-underlinenav-icon: #6e7681; --color-underlinenav-border-hover: rgba(110,118,129,0.4); --color-action-list-item-inline-divider: rgba(48,54,61,0.48); --color-action-list-item-default-hover-bg: rgba(177,186,196,0.12); --color-action-list-item-default-hover-border: rgba(0,0,0,0); --color-action-list-item-default-active-bg: rgba(177,186,196,0.2); --color-action-list-item-default-active-border: rgba(0,0,0,0); --color-action-list-item-default-selected-bg: rgba(177,186,196,0.08); --color-action-list-item-danger-hover-bg: rgba(248,81,73,0.16); --color-action-list-item-danger-active-bg: rgba(248,81,73,0.24); --color-action-list-item-danger-hover-text: #ff7b72; --color-switch-track-bg: rgba(110,118,129,0.1); --color-switch-track-hover-bg: hsla(215,8%,72%,0.1); --color-switch-track-active-bg: rgba(110,118,129,0.4); --color-switch-track-disabled-bg: #21262d; --color-switch-track-fg: #7d8590; --color-switch-track-disabled-fg: #010409; --color-switch-track-border: rgba(0,0,0,0); --color-switch-track-checked-bg: rgba(31,111,235,0.35); --color-switch-track-checked-hover-bg: rgba(31,111,235,0.5); --color-switch-track-checked-active-bg: rgba(31,111,235,0.65); --color-switch-track-checked-fg: #ffffff; --color-switch-track-checked-disabled-fg: #010409; --color-switch-track-checked-border: rgba(0,0,0,0); --color-switch-knob-bg: #0d1117; --color-switch-knob-disabled-bg: #161b22; --color-switch-knob-border: #606771; --color-switch-knob-checked-bg: #0d1117; --color-switch-knob-checked-disabled-bg: #161b22; --color-switch-knob-checked-border: rgba(31,111,235,0.35); --color-segmented-control-bg: rgba(110,118,129,0.1); --color-segmented-control-button-bg: #0d1117; --color-segmented-control-button-hover-bg: #30363d; --color-segmented-control-button-active-bg: #21262d; --color-segmented-control-button-selected-border: #6e7681; --color-tree-view-item-chevron-hover-bg: rgba(177,186,196,0.12); --color-tree-view-item-directory-fill: #7d8590; --color-fg-default: #e6edf3; --color-fg-muted: #7d8590; --color-fg-subtle: #6e7681; --color-fg-on-emphasis: #ffffff; --color-canvas-default: #0d1117; --color-canvas-overlay: #161b22; --color-canvas-inset: #010409; --color-canvas-subtle: #161b22; --color-border-default: #30363d; --color-border-muted: #21262d; --color-border-subtle: rgba(240,246,252,0.1); --color-shadow-small: 0 0 transparent; --color-shadow-medium: 0 3px 6px #010409; --color-shadow-large: 0 8px 24px #010409; --color-shadow-extra-large: 0 12px 48px #010409; --color-neutral-emphasis-plus: #6e7681; --color-neutral-emphasis: #6e7681; --color-neutral-muted: rgba(110,118,129,0.4); --color-neutral-subtle: rgba(110,118,129,0.1); --color-accent-fg: #2f81f7; --color-accent-emphasis: #1f6feb; --color-accent-muted: rgba(56,139,253,0.4); --color-accent-subtle: rgba(56,139,253,0.1); --color-success-fg: #3fb950; --color-success-emphasis: #238636; --color-success-muted: rgba(46,160,67,0.4); --color-success-subtle: rgba(46,160,67,0.15); --color-attention-fg: #d29922; --color-attention-emphasis: #9e6a03; --color-attention-muted: rgba(187,128,9,0.4); --color-attention-subtle: rgba(187,128,9,0.15); --color-severe-fg: #db6d28; --color-severe-emphasis: #bd561d; --color-severe-muted: rgba(219,109,40,0.4); --color-severe-subtle: rgba(219,109,40,0.1); --color-danger-fg: #f85149; --color-danger-emphasis: #da3633; --color-danger-muted: rgba(248,81,73,0.4); --color-danger-subtle: rgba(248,81,73,0.1); --color-open-fg: #3fb950; --color-open-emphasis: #238636; --color-open-muted: rgba(46,160,67,0.4); --color-open-subtle: rgba(46,160,67,0.15); --color-closed-fg: #f85149; --color-closed-emphasis: #da3633; --color-closed-muted: rgba(248,81,73,0.4); --color-closed-subtle: rgba(248,81,73,0.15); --color-done-fg: #a371f7; --color-done-emphasis: #8957e5; --color-done-muted: rgba(163,113,247,0.4); --color-done-subtle: rgba(163,113,247,0.1); --color-sponsors-fg: #db61a2; --color-sponsors-emphasis: #bf4b8a; --color-sponsors-muted: rgba(219,97,162,0.4); --color-sponsors-subtle: rgba(219,97,162,0.1); --color-primer-fg-disabled: #484f58; --color-primer-canvas-backdrop: rgba(1,4,9,0.8); --color-primer-canvas-sticky: rgba(13,17,23,0.95); --color-primer-border-active: #f78166; --color-primer-border-contrast: rgba(255,255,255,0.2); --color-primer-shadow-highlight: 0 0 transparent; --color-primer-shadow-inset: 0 0 transparent; --color-scale-black: #010409; --color-scale-white: #ffffff; --color-scale-gray-0: #f0f6fc; --color-scale-gray-1: #c9d1d9; --color-scale-gray-2: #b1bac4; --color-scale-gray-3: #8b949e; --color-scale-gray-4: #6e7681; --color-scale-gray-5: #484f58; --color-scale-gray-6: #30363d; --color-scale-gray-7: #21262d; --color-scale-gray-8: #161b22; --color-scale-gray-9: #0d1117; --color-scale-blue-0: #cae8ff; --color-scale-blue-1: #a5d6ff; --color-scale-blue-2: #79c0ff; --color-scale-blue-3: #58a6ff; --color-scale-blue-4: #388bfd; --color-scale-blue-5: #1f6feb; --color-scale-blue-6: #1158c7; --color-scale-blue-7: #0d419d; --color-scale-blue-8: #0c2d6b; --color-scale-blue-9: #051d4d; --color-scale-green-0: #aff5b4; --color-scale-green-1: #7ee787; --color-scale-green-2: #56d364; --color-scale-green-3: #3fb950; --color-scale-green-4: #2ea043; --color-scale-green-5: #238636; --color-scale-green-6: #196c2e; --color-scale-green-7: #0f5323; --color-scale-green-8: #033a16; --color-scale-green-9: #04260f; --color-scale-yellow-0: #f8e3a1; --color-scale-yellow-1: #f2cc60; --color-scale-yellow-2: #e3b341; --color-scale-yellow-3: #d29922; --color-scale-yellow-4: #bb8009; --color-scale-yellow-5: #9e6a03; --color-scale-yellow-6: #845306; --color-scale-yellow-7: #693e00; --color-scale-yellow-8: #4b2900; --color-scale-yellow-9: #341a00; --color-scale-orange-0: #ffdfb6; --color-scale-orange-1: #ffc680; --color-scale-orange-2: #ffa657; --color-scale-orange-3: #f0883e; --color-scale-orange-4: #db6d28; --color-scale-orange-5: #bd561d; --color-scale-orange-6: #9b4215; --color-scale-orange-7: #762d0a; --color-scale-orange-8: #5a1e02; --color-scale-orange-9: #3d1300; --color-scale-red-0: #ffdcd7; --color-scale-red-1: #ffc1ba; --color-scale-red-2: #ffa198; --color-scale-red-3: #ff7b72; --color-scale-red-4: #f85149; --color-scale-red-5: #da3633; --color-scale-red-6: #b62324; --color-scale-red-7: #8e1519; --color-scale-red-8: #67060c; --color-scale-red-9: #490202; --color-scale-purple-0: #eddeff; --color-scale-purple-1: #e2c5ff; --color-scale-purple-2: #d2a8ff; --color-scale-purple-3: #bc8cff; --color-scale-purple-4: #a371f7; --color-scale-purple-5: #8957e5; --color-scale-purple-6: #6e40c9; --color-scale-purple-7: #553098; --color-scale-purple-8: #3c1e70; --color-scale-purple-9: #271052; --color-scale-pink-0: #ffdaec; --color-scale-pink-1: #ffbedd; --color-scale-pink-2: #ff9bce; --color-scale-pink-3: #f778ba; --color-scale-pink-4: #db61a2; --color-scale-pink-5: #bf4b8a; --color-scale-pink-6: #9e3670; --color-scale-pink-7: #7d2457; --color-scale-pink-8: #5e103e; --color-scale-pink-9: #42062a; --color-scale-coral-0: #ffddd2; --color-scale-coral-1: #ffc2b2; --color-scale-coral-2: #ffa28b; --color-scale-coral-3: #f78166; --color-scale-coral-4: #ea6045; --color-scale-coral-5: #cf462d; --color-scale-coral-6: #ac3220; --color-scale-coral-7: #872012; --color-scale-coral-8: #640d04; --color-scale-coral-9: #460701; --base-size-4: 0.25rem; --base-size-8: 0.5rem; --base-size-12: 0.75rem; --base-size-16: 1rem; --base-size-20: 1.25rem; --base-size-24: 1.5rem; --base-size-28: 1.75rem; --base-size-32: 2rem; --base-size-36: 2.25rem; --base-size-40: 2.5rem; --base-size-44: 2.75rem; --base-size-48: 3rem; --base-size-64: 4rem; --base-size-80: 5rem; --base-size-96: 6rem; --base-size-112: 7rem; --base-size-128: 8rem; --base-text-weight-light: 300; --base-text-weight-normal: 400; --base-text-weight-medium: 500; --base-text-weight-semibold: 600; --boxShadow-thin: inset 0 0 0 max(1px, 0.0625rem); --boxShadow-thick: inset 0 0 0 max(2px, 0.125rem); --boxShadow-thicker: inset 0 0 0 max(4px, 0.25rem); --borderWidth-thin: max(1px, 0.0625rem); --borderWidth-thick: max(2px, 0.125rem); --borderWidth-thicker: max(4px, 0.25rem); --borderRadius-small: 0.1875rem; --borderRadius-medium: 0.375rem; --borderRadius-large: 0.75rem; --borderRadius-full: 624.9375rem; --outline-focus-offset: -0.125rem; --outline-focus-width: 0.125rem; --breakpoint-xsmall: 20rem; --breakpoint-small: 34rem; --breakpoint-medium: 48rem; --breakpoint-large: 63.25rem; --breakpoint-xlarge: 80rem; --breakpoint-xxlarge: 90rem; --control-minTarget-auto: 2.75rem; --controlStack-small-gap-auto: 1rem; --controlStack-medium-gap-auto: 0.75rem; --control-minTarget-fine: 1rem; --control-minTarget-coarse: 2.75rem; --control-xsmall-size: 1.5rem; --control-xsmall-lineBoxHeight: 1.25rem; --control-xsmall-paddingBlock: 0.125rem; --control-xsmall-paddingInline-condensed: 0.25rem; --control-xsmall-paddingInline-normal: 0.5rem; --control-xsmall-paddingInline-spacious: 0.75rem; --control-xsmall-gap: 0.25rem; --control-small-size: 1.75rem; --control-small-lineBoxHeight: 1.25rem; --control-small-paddingBlock: 0.25rem; --control-small-paddingInline-condensed: 0.5rem; --control-small-paddingInline-normal: 0.75rem; --control-small-gap: 0.25rem; --control-medium-size: 2rem; --control-medium-lineBoxHeight: 1.25rem; --control-medium-paddingBlock: 0.375rem; --control-medium-paddingInline-condensed: 0.5rem; --control-medium-paddingInline-normal: 0.75rem; --control-medium-paddingInline-spacious: 1rem; --control-medium-gap: 0.5rem; --control-large-size: 2.5rem; --control-large-lineBoxHeight: 1.25rem; --control-large-paddingBlock: 0.625rem; --control-large-paddingInline-normal: 0.75rem; --control-large-paddingInline-spacious: 1rem; --control-large-gap: 0.5rem; --control-xlarge-size: 3rem; --control-xlarge-lineBoxHeight: 1.25rem; --control-xlarge-paddingBlock: 0.875rem; --control-xlarge-paddingInline-normal: 0.75rem; --control-xlarge-paddingInline-spacious: 1rem; --control-xlarge-gap: 0.5rem; --controlStack-small-gap-condensed: 0.5rem; --controlStack-small-gap-spacious: 1rem; --controlStack-medium-gap-condensed: 0.5rem; --controlStack-medium-gap-spacious: 0.75rem; --controlStack-large-gap-auto: 0.5rem; --controlStack-large-gap-condensed: 0.5rem; --controlStack-large-gap-spacious: 0.75rem; --space-xxsmall: 0.125rem; --space-xsmall: 0.25rem; --space-small: 0.375rem; --space-medium: 0.5rem; --space-large: 0.75rem; --space-xlarge: 1rem; --stack-padding-condensed: 0.5rem; --stack-padding-normal: 1rem; --stack-padding-spacious: 1.5rem; --stack-gap-condensed: 0.5rem; --stack-gap-normal: 1rem; --stack-gap-spacious: 1.5rem; --overlay-width-xsmall: 12rem; --overlay-width-small: 20rem; --overlay-width-medium: 30rem; --overlay-width-large: 40rem; --overlay-width-xlarge: 60rem; --overlay-height-small: 16rem; --overlay-height-medium: 20rem; --overlay-height-large: 27rem; --overlay-height-xlarge: 37.5rem; --overlay-padding-normal: 1rem; --overlay-padding-condensed: 0.5rem; --overlay-paddingBlock-condensed: 0.25rem; --overlay-paddingBlock-normal: 0.75rem; --overlay-borderRadius: 0.375rem; --text-codeInline-size: 0.9285em; --text-codeBlock-lineHeight: 1.5385; --text-codeBlock-size: 0.8125rem; --text-caption-lineHeight: 1.3333; --text-caption-size: 0.75rem; --text-body-lineHeight-small: 1.6666; --text-body-lineHeight-medium: 1.4285; --text-body-lineHeight-large: 1.5; --text-body-size-small: 0.75rem; --text-body-size-medium: 0.875rem; --text-body-size-large: 1rem; --text-subtitle-lineHeight: 1.6; --text-subtitle-size: 1.25rem; --text-title-lineHeight-small: 1.5; --text-title-lineHeight-medium: 1.6; --text-title-lineHeight-large: 1.5; --text-title-size-small: 1rem; --text-title-size-medium: 1.25rem; --text-title-size-large: 2rem; --text-display-lineHeight: 1.4; --text-display-size: 2.5rem; --text-display-lineBoxHeight: 3.5rem; --fontStack-monospace: ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace; --fontStack-sansSerif: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Noto Sans', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'; --fontStack-system: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Noto Sans', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'; --text-codeInline-weight: 400; --text-codeBlock-weight: 400; --text-caption-weight: 400; --text-body-weight: 400; --text-subtitle-weight: 400; --text-title-weight-small: 600; --text-title-weight-medium: 600; --text-title-weight-large: 600; --text-display-weight: 500; --text-codeInline-shorthand: var(--text-codeInline-weight) var(--text-codeInline-size) var(--fontStack-monospace); --text-codeBlock-shorthand: var(--text-codeBlock-weight) var(--text-codeBlock-size)/var(--text-codeBlock-lineHeight) var(--fontStack-monospace); --text-caption-shorthand: var(--text-caption-weight) var(--text-caption-size)/var(--text-caption-lineHeight) var(--fontStack-sansSerif); --text-body-shorthand-small: var(--text-body-weight) var(--text-body-size-small)/var(--text-body-lineHeight-small) var(--fontStack-sansSerif); --text-body-shorthand-medium: var(--text-body-weight) var(--text-body-size-medium)/var(--text-body-lineHeight-medium) var(--fontStack-sansSerif); --text-body-shorthand-large: var(--text-body-weight) var(--text-body-size-large)/var(--text-body-lineHeight-large) var(--fontStack-sansSerif); --text-subtitle-shorthand: var(--text-subtitle-weight) var(--text-subtitle-size)/var(--text-subtitle-lineHeight) var(--fontStack-sansSerif); --text-title-shorthand-small: var(--text-title-weight-small) var(--text-title-size-small)/var(--text-title-lineHeight-small) var(--fontStack-sansSerif); --text-title-shorthand-medium: var(--text-title-weight-medium) var(--text-title-size-medium)/var(--text-title-lineHeight-medium) var(--fontStack-sansSerif); --text-title-shorthand-large: var(--text-title-weight-large) var(--text-title-size-large)/var(--text-title-lineHeight-large) var(--fontStack-sansSerif); --text-display-shorthand: var(--text-display-weight) var(--text-display-size)/var(--text-display-lineHeight) var(--fontStack-sansSerif); color-scheme: dark; color: var(--fgColor-default, var(--color-fg-default)); background-color: var(--bgColor-default, var(--color-canvas-default)); --Layout-pane-width: 220px; --Layout-content-width: 100%; --Layout-template-columns: 1fr var(--Layout-pane-width); --Layout-template-areas: "content pane"; --Layout-column-gap: 16px; --Layout-row-gap: 16px; --Layout-outer-spacing-x: 0px; --Layout-outer-spacing-y: 0px; --Layout-inner-spacing-min: 0px; --Layout-inner-spacing-max: 0px; --actionListContent-paddingBlock: var(--control-medium-paddingBlock,0.375rem); --duration-fast: 80ms; --easing-easeInOut: cubic-bezier(0.65,0,0.35,1); --color-social-reaction-bg-hover: var(--color-scale-gray-7); --color-social-reaction-bg-reacted-hover: var(--color-scale-blue-8); --color-workflow-card-connector: var(--color-scale-gray-5); --color-workflow-card-connector-bg: var(--color-scale-gray-5); --color-workflow-card-connector-inactive: var(--borderColor-default, var(--color-border-default)); --color-workflow-card-connector-inactive-bg: var(--borderColor-default, var(--color-border-default)); --color-workflow-card-connector-highlight: var(--color-scale-blue-5); --color-workflow-card-connector-highlight-bg: var(--color-scale-blue-5); --color-workflow-card-bg: var(--color-scale-gray-7); --color-workflow-card-inactive-bg: var(--bgColor-inset, var(--color-canvas-inset)); --color-workflow-card-header-shadow: rgba(27, 31, 35, 0.04); --color-workflow-card-progress-complete-bg: var(--color-scale-blue-5); --color-workflow-card-progress-incomplete-bg: var(--color-scale-gray-6); --color-discussions-state-answered-icon: var(--color-scale-green-3); --color-bg-discussions-row-emoji-box: var(--color-scale-gray-6); --color-notifications-button-text: var(--color-scale-white); --color-notifications-button-hover-text: var(--color-scale-white); --color-notifications-button-hover-bg: var(--color-scale-blue-4); --color-notifications-row-read-bg: var(--color-canvas-default); --color-notifications-row-bg: var(--bgColor-muted, var(--color-canvas-subtle)); --color-icon-directory: var(--fgColor-muted, var(--color-fg-muted)); --color-checks-step-error-icon: var(--color-scale-red-4); --color-calendar-halloween-graph-day-L1-bg: #631c03; --color-calendar-halloween-graph-day-L2-bg: #bd561d; --color-calendar-halloween-graph-day-L3-bg: #fa7a18; --color-calendar-halloween-graph-day-L4-bg: #fddf68; --color-calendar-winter-graph-day-L1-bg: #0A3069; --color-calendar-winter-graph-day-L2-bg: #0969DA; --color-calendar-winter-graph-day-L3-bg: #54AEFF; --color-calendar-winter-graph-day-L4-bg: #B6E3FF; --color-calendar-graph-day-bg: var(--color-scale-gray-8); --color-calendar-graph-day-border: rgba(27, 31, 35, 0.06); --color-calendar-graph-day-L1-bg: #0e4429; --color-calendar-graph-day-L2-bg: #006d32; --color-calendar-graph-day-L3-bg: #26a641; --color-calendar-graph-day-L4-bg: #39d353; --color-calendar-graph-day-L1-border: rgba(255, 255, 255, 0.05); --color-calendar-graph-day-L2-border: rgba(255, 255, 255, 0.05); --color-calendar-graph-day-L3-border: rgba(255, 255, 255, 0.05); --color-calendar-graph-day-L4-border: rgba(255, 255, 255, 0.05); --color-user-mention-fg: var(--color-scale-yellow-0); --color-user-mention-bg: var(--color-scale-yellow-8); --color-dashboard-feed-bg: var(--color-scale-gray-9); --color-text-white: var(--color-scale-white); --line-number-cell-width: 40px; --diff-line-minimum-height: 24px; --border-color-iteration-1: #363940; --border-color-iteration-2: #3F434B; --border-color-iteration-3: #4B5159; --control-borderColor-rest: var(--border-color-iteration-4, var(--border-color-iteration-3, var(--border-color-iteration-2, var(--border-color-iteration-1)))) !important;">

2.5 is now available. This now includes logging for both the app and the installer. You could install and set the main app log level, in Preferences, to "Debug". Providing the logs will help to see what's going on.

Write Preview

Attach files by dragging & dropping, selecting or pasting them.

 Close with comment

Comment

Remember, contributions to this repository should follow our GitHub Community Guidelines.

AssigneesNo one—assign yourself

Labels

None yet

Projects

None yet

Milestone

DevelopmentCreate a branch for this issue or link a pull request.

Notifications

Customize

Unsubscribe

You’re receiving notifications because you’re watching this repository.

2 participants

 Lock conversation

 Pin issue 

 Transfer issue

Convert to discussion

 Delete issue

Footer

© 2023 GitHub, Inc.

Footer navigationTermsPrivacySecurityStatusDocsContact GitHubPricingAPITrainingBlogAbout

Owner avalon60 commented 2.5 is now available. This now includes logging for both the app and the installer. You could install and set the main app log level, in Preferences, to "Debug". Providing the logs will help to see what's going on.

**Comment by avalon60** [comment]
Closing - no response.
